node-scripts/src/pipeline3/scripts.js

```diff
@@ -154,8 +154,9 @@ module.exports = {
     return script;
   },
 
-  initFolder: function(folderPath, user, wipe) {
-    var group = user;
+  initFolder: function(folderPath, user, wipe, group) {
```

Comment:
@ivange94 , I think best is to group "user" and "group" together as function parameters:
```
function(folderPath, user, group, wipe) {
```

Reply:
That's what I initially did and that makes more sense but I feel it's a bad practice modifying parameter order since the 4rd parameter is already expected to be boolean. I did it that way for backwards compatibility.

Reply:
Good point. But in our case, we can easily identify all places where this method is called and modify it. This method is not used in any other project or anything so that is absolutely safe to change the params orders. This would definitely apply in the case of a public API. Here it is just a utility method that we have full control on.

Reply:
Ok